CHANGELOG.md

```diff
@@ -2,12 +2,17 @@
 
 ## Unreleased Changes
 
+### FEATURES
+- [modules]
+  -Implementation of the `ChanOpenAck`, `ChanOpenConfirm`, `ChanCloseInit`, and `ChanCloseConfirm` handlers ([#316])
```

Comment:
```suggestion
  - Implementation of the `ChanOpenAck`, `ChanOpenConfirm`, `ChanCloseInit`, and `ChanCloseConfirm` handlers ([#316])
```

WIthout the additional empty space after the dash, this will not render as a bullet list.